### DISCUSSION TITLE
Blog social media share buttons

### DISCUSSION THREAD

**Discussion by reda-jaifar** [facebook/docusaurus#10136] [Ideas]
I would like to have a social media share buttons ( linkedin, twitter, etc ...) to share a blog post

**Comment by slorber** [comment]
Yes, we'll introduce that soon

As part of the blog authors pages feature (https://github.com/facebook/docusaurus/issues/9825), we'll need to use the author's name link to be the link to the author page.

So we also need another way to link to author URL(s) and social profiles, and having extra social icons should be a good first step in that direction that we could ship earlier.

I found this nice resource with useful logos we could use: https://svgl.app/

cc @OzakIOne
